rosetta-lang/src/main/java/com/regnosys/rosetta/types/builtin/RBuiltinTypeService.java

```diff
@@ -81,7 +78,18 @@ public Optional<LinkedHashMap<String, RosettaValue>> reverse(RType type) {
 	public final RBasicType PATTERN = registerConstantType(new RBasicType("pattern", false));
 	public final RMetaAnnotatedType PATTERN_WITH_NO_META = RMetaAnnotatedType.withNoMeta(PATTERN);
 	public final RBasicType NOTHING = registerConstantType(new RBasicType("nothing", true));
-	public final RMetaAnnotatedType NOTHING_WITH_NO_META = RMetaAnnotatedType.withNoMeta(NOTHING);
+	public final RMetaAnnotatedType NOTHING_WITH_ANY_META = new RMetaAnnotatedType(NOTHING, List.of()) {
+		@Override
+		public List<RMetaAttribute> getMetaAttributes() {
+			StringBuilder sb = new StringBuilder();
+			StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
+			for (StackTraceElement stackTraceElement : stackTrace) {
+				sb.append("\tat ").append(stackTraceElement).append("\n");
+			}
+			LOGGER.error("getMetaAttributes called on type NOTHING_WITH_ANY_META:\n{}", sb);
```

Comment:
Doesn't the logger automatically include a stack trace if you do `LOGGER.error(..., new Exception())`?